ledger-service/http-json/src/main/scala/com/digitalasset/http/ContractsFetch.scala

```diff
@@ -188,6 +189,13 @@ private class ContractsFetch(
       contractId = ac.contractId.unwrap,
       templateId = ac.templateId,
       key = lfKey.cata(lfValueToDbJsValue, JsNull),
+      keyHash = lfKey.cata(
+        Hash
+          .assertHashContractKey(TemplateId.toLedgerApiValue(ac.templateId), _)
+          .toHexString
+          .toString,
+        "",
```

Comment:
not sure about the `""` string as default value

Reply:
Maybe `JsNull`?